client/components/environment-badge/index.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+import { string, node } from 'prop-types';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+
+export function PreferencesHelper() {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+	return <div className="environment is-prefs" />;
+	/* eslint-enable wpcalypso/jsx-classname-namespace */
```

Comment:
You could do `// eslint-enable-line wpcalypso/jsx-classname-namespace` on the same line as the offending `className`.

Reply:
(Same in a few more instances below.)

Reply:
Actually -- why not change the `className`s (and corresponding styling) to comply with `eslint`'s suggestion (i.e. `environment-badge`)? 😄